utils/generator/transformers/enums_producer.py

```diff
@@ -0,0 +1,118 @@
+"""
+Enums transformation
+"""
+
+import logging
+import textwrap
+from collections import namedtuple, OrderedDict
+
+from model.enum import Enum
+from model.enum_element import EnumElement
+from transformers.common_producer import InterfaceProducerCommon
+
+
+class EnumsProducer(InterfaceProducerCommon):
+    """
+    Enums transformation
+    """
+
+    def __init__(self, paths):
+        super(EnumsProducer, self).__init__(
+            container_name='elements',
+            enums_package=None,
+            structs_package=None,
+            package_name=paths.enums_package)
+        self.logger = logging.getLogger('EnumsProducer')
+        self._params = namedtuple('params', 'origin name internal description since value deprecated')
+
+    @staticmethod
+    def converted(name):
+        if name[0].isdigit():
+            name = '_' + name
+        if '-' in name:
+            name = name.replace('-', '_')
+        return name
+
+    def transform(self, item: Enum) -> dict:
+        """
+        Override
+        :param item: particular element from initial Model
+        :return: dictionary to be applied to jinja2 template
+        """
+        imports = set()
+        params = OrderedDict()
+        if any(map(lambda l: l.name != self.converted(l.name) and getattr(l, 'value', None) is not None,
+                   getattr(item, self.container_name).values())):
+            kind = 'complex'
+            imports.add('java.util.EnumSet')
+        elif any(map(lambda l:
+                     l.name != self.converted(l.name) or l.name[0].isdigit() or getattr(l, 'value', None) is not None,
```

Comment:
do you think we can remove this `l.name[0].isdigit()` from the condition since it is already covered in `l.name != self.converted(l.name)`?

Reply:
yes, you right. 
Deleting it.